src/System.Devices.Gpio/System.Devices.Gpio/GPIOPin.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Devices.Gpio
+{
+    public enum GpioPinMode
+    {
+        Input = 0,
+        Output = 1
+    }
+
+    public enum GpioPinValue
+    {
+        Low = 0,
+        High = 1
+    }
+
+    public enum GpioScheme
+    {
+        Board,
+        BCM
+    }
+
+    public class GpioPin
```

Comment:
we should consider if it's possible to drop the redundant Gpio prefix on all types or whether it's necessary to keep it due to conflicts with other well-known types or keywords.

Reply:
Gpio prefix removed.